internal/overlord/servstate/manager_test.go

```diff
@@ -70,7 +70,7 @@ var planLayer1 = `
 services:
     test1:
         override: replace
-        command: /bin/sh -c "echo test1 | tee -a %s; sleep 300"
+        command: /bin/sh -c "echo test1 | tee -a %s; sleep 10"
```

Comment:
Drive-by fix to reduce the unnecessarily sleep time in these tests. None of them are alive for more than a few hundred milliseconds, so 10s is ample.